src/components/layout.js

```diff
@@ -68,37 +68,43 @@ class Layout extends React.Component {
   }
 
   selectOrganization = orgId => {
-    this.props.dispatch(organizationSelect(orgId));
+    const { dispatch } = this.props;
+
+    dispatch(organizationSelect(orgId));
+    dispatch(push('/'));
```

Comment:
Is there any reason why you are pushing after dispatching instead of adding the `'/'` in the return`getOrganizationURL()`?

Reply:
In the previous implementation, the `push` was in the `organizationSelect` action creator. It kind of makes sense, since when selecting an organization from the dropdown in the header, you could be on a page that might not be available for another organization, so it just directs you to the homepage, to avoid any unwanted errors.

The reason for not leaving it in `organizationSelect` is that it's only needed when changing organizations from the header dropdown, otherwise not. Hope that makes sense 😁

Reply:
Totally, thanks!